seeding-DMLs/SeedingBooks.sql

```diff
@@ -0,0 +1,33 @@
+USE [LibraryManagement]
+GO
+
+-- Insert 1,000 rows of fictional data into the Books table
+DECLARE @Counter INT = 1;
+
+WHILE @Counter <= 1000
+BEGIN
+    INSERT INTO [dbo].[Books] ([BookID], [Title], [Author], [ISBN], [PublishedDate], [Genre], [ShelfLocation], [CurrentStatus])
+    VALUES 
+    (
+        @Counter,
+        LEFT(NEWID(), 10), 
+        LEFT(NEWID(), 10), 
+        ABS(CHECKSUM(NEWID())) % 1000000000000, 
```

Comment:
You can use the `FORMAT` function to improve data clarity. Since an ISBN typically 13 digits
`FORMAT(ABS(CHECKSUM(NEWID())) % 1000000000000, '0000000000000')`.